deployer/deployer.py

```diff
@@ -408,21 +416,24 @@ def delete_asg(self, asg_name):
         r = self.request("cluster/delete", data)
         return r.status_code == 200
 
-    def create_loadbalancer(self):
+    def create_loadbalancer(self, health_check, health_check_port):
         data = {
             'appName': self.app,
             'selectedZones': ['eu-west-1a', 'eu-west-1b', 'eu-west-1c'],
             'selectedSecurityGroups': self.security_group,
-            'protocol1': 'HTTP',
-            'lbPort1': '80',
-            'instancePort1': '8000',
             'subnetPurpose': 'external',
-            'target': 'HTTP:8000/healthcheck',
+            'target': 'HTTP:%s/%s' % (health_check_port, health_check),
```

Comment:
@padilo although this is correct is better use the new way:
```
'target': 'HTTP:{0}/{1}'.format(health_check_port, health_check),
```
https://pyformat.info/